lib/paypalhttp/serializers/form_encoded.rb

```diff
@@ -1,11 +1,15 @@
-require 'cgi'
+require 'uri'
 
 module PayPalHttp
   class FormEncoded
+    def initialize
+      @parser = parser = URI::Parser.new()
```

Comment:
Is the double assignment needed here? Shouldn't it be:
```suggestion
      @parser = URI::Parser.new()
```

Reply:
I agree, this must've just been a typo when moving the declaration into the initializer